### ISSUE TITLE
Insertion with parameters

### ISSUE THREAD

**Issue by Juergen-aus-Zuendorf** [turnermm/ckgdoku#5]
Hi,

when inserting an image with parameters the import brings out an error message:

![2017-03-28 08_53_24-dateiauswahl schutte-wiki](https://cloud.githubusercontent.com/assets/1800472/24392503/75e16116-1394-11e7-9a95-52af464b52e4.png)

![2017-03-28 08_53_42- spielplatz schutte-wiki](https://cloud.githubusercontent.com/assets/1800472/24392511/80682c78-1394-11e7-9529-33d88e41bc98.png)

![2017-03-28 08_57_41- spielplatz schutte-wiki](https://cloud.githubusercontent.com/assets/1800472/24392546/ac4b9622-1394-11e7-8704-fb8569c7fdbe.png)

Best regards 
Juergen

**Comment by turnermm** [comment]
What is the translation of nolin?

**Comment by turnermm** [comment]
Right -- I see the problem.  "nolin" is from "nolink".  I don't use ckgdoku myself and I didn't code the connection between dokuwiki's media manager and ckgedit. But I did notice that the parameters were not always being inserted, though I never saw the "nolin" insertion.  Anyway, it should now be fixed.  Also, please see issue: https://github.com/turnermm/ckgdoku/issues/4.  The transition that I refer to there is not complete.

**Comment by turnermm** [comment]
I was working on my tablet and my finger accidentally hit the delete button for the original reply to this.  Anyway, in case you hadn't seen it earlier, I've fixed the problem.  "nolin" is from the buggy insertion of "nolink" into the size box instead of into the link type box.  As I explained earlier I didn't write the code that ties the media manager to ckgedit and I don't use ckgdoku as a regular thing, so I had never noticed this    error myself.

Also, see https://github.com/turnermm/ckgdoku/issues/4.
